.github/workflows/publish_debian.yml

```diff
@@ -0,0 +1,43 @@
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'base/debian/*'
+  pull_request:
```

Comment:
I don't think this should run on pull requests, just pushes to `main`. Otherwise a pull request into `main` will publish a new image.

Reply:
It'll only push when code is commited to main. [click here](https://github.com/wyzlle/nixpacks/blob/main/.github/workflows/publish_debian.yml#L42)

Reply:
Ah I see, missed that.